notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,2260 @@
+{
```

Comment:
<p>3) (Local Graph) a manifold can be understood as a d-dimensional surface described by d variables, i.e. by the "graph" of a smooth function $f: (x_1, ..., x_d) \rightarrow f(x, y)$ for the d variables x_1, ..., x_d. See (a) in figure below.</p><p>2) (Local Implicit Function) a manifold can be understood as the set of points that verify a constraint defined by an implicit equation, given some function f, called an implicit function. See (b) in figure below.</p><p>1) (Local Parametrization)  ... see (c) in figure below.</p><p> </p><p>NIT: Euclidean is the convention we use to refer to the Euclidean/Euclidian space in geomstats (I know people use both writings).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>done</p>